### PR TITLE
Remove alignment hack

### PR DESCRIPTION
**Pull request by yifanlu**
Let's discuss why the alignment hack is there and if we can rid it

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+script:
+ - export REV=dirty-$(git describe --always)
+ - chmod +x *.sh
+ - ./build.sh
+before_deploy:
+  - git config --global user.email "builds@travis-ci.com"
+  - git config --global user.name "Travis CI"
+  - export GIT_TAG=$TRAVIS_BRANCH-$TRAVIS_OS_NAME-v$TRAVIS_BUILD_NUMBER
+  - git tag $GIT_TAG -a -m "Generated tag from TravisCI for build $TRAVIS_BUILD_NUMBER"
+  - git push -q https://$TAGPERM@github.com/frangarcj/buildscripts --tags
```
Comment (d3m3vilurr) [inline .travis.yml]: `frangarcj/buildscripts` should change $TRAVIS_REPO_SLUG

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+deploy:
+  provider: releases
+  api_key: $OAUTH
+  file_glob: true
+  file: "build/*.tar.bz2"
+  skip_cleanup: true
+  on:
+    tags: false
+    all_branches: true
```
Comment (d3m3vilurr) [inline .travis.yml]: how about only deploy with tags or just master branch?

**Comment by yifanlu** [comment]
`vita-elf-create: elf_update(dest, ELF_C_WRITE) >= 0 failed: Layout error: overlapping sections`

Maybe this is the problem

**Comment by d3m3vilurr** [comment]
can you split pr to `add travis` & `remove hack`?

**Comment by xyzz** [comment]
The alignment hack was added because otherwise data/bss was not aligned at all. So if you wrote attribute(aligned) thing it would only be aligned if you get lucky and your text segment size is good.

**Comment by yifanlu** [comment]
How is that not a problem with gcc and friends?

**Comment by xyzz** [comment]
I haven't tested it, it could be a problem elsewhere too.

**Comment by xyzz** [comment]
This pull is broken, I'm closing it.

**Pull request closed without merging**